hubblestack/extmods/modules/nebula_osquery.py

```diff
@@ -74,11 +74,27 @@ def queries(query_group,
         salt '*' nebula.queries hour verbose=True
         salt '*' nebula.queries hour pillar_key=sec_osqueries
     '''
+    query_data = {}
+
     if query_file is None:
         if salt.utils.is_windows():
             query_file = 'salt://hubblestack_nebula/hubblestack_nebula_win_queries.yaml'
         else:
             query_file = 'salt://hubblestack_nebula/hubblestack_nebula_queries.yaml'
+
+    if not isinstance(query_file, list):
+        query_file = [query_file]
+
+    for file in query_file:
```

Comment:
Can you use a different variable name than `file`? `file` is a reserved built-in in python and is being [shadowed](https://en.wikipedia.org/wiki/Variable_shadowing) here.